Tilers/CityTiler/database_accesses.py

```diff
@@ -70,23 +83,21 @@ def open_data_bases(db_config_file_paths):
     return cursors
 
 
-def get_buildings_from_3dcitydb(cursor, buildings=None):
+def get_buildings_objects_from_3dcitydb(cursor, buildings):
```

Comment:
If we follow the logic of converting the code from being function oriented to object
oriented, could we make this get_buildings_objects_from_3dcitydb() function a member
function of the CityMBuildings class (to be trivially created in citym_building.py as 
inheritance alias of CityMObjects class) ?